justfile

```diff
@@ -5,8 +5,23 @@ default:
     @just --list
 
 # Update the OpenApi yml files by generating these from the tAPIr specs
-alias dog := docs-openapi-generate
 docs-openapi-generate:
-    mkdir -p {{openapiDir}}
-    rm {{openapiDir}}/*.yml >> /dev/null 2>&1 || true
-    sbt "webapi/runMain org.knora.webapi.slice.common.api.DocsGenerator {{openapiDir}}"
+    mkdir -p {{ openapiDir }}
+    rm {{ openapiDir }}/*.yml >> /dev/null 2>&1 || true
+    sbt "webapi/runMain org.knora.webapi.slice.common.api.DocsGenerator {{ openapiDir }}"
+
+alias dog := docs-openapi-generate
+
+# Start stack from latest images on dockerhub (no build required)
+start-stack-latest:
+    docker compose down
+    docker compose pull
+    docker compose up -d
+
+alias ssl := start-stack-latest
```

Comment:
Maybe move all the aliases to the top of the file so we have an overview of which aliases are already configured?